tangle-compiler/src/main/kotlin/tangle/inject/compiler/descriptor.kt

```diff
@@ -145,7 +146,7 @@ fun TypeParameterDescriptor.boundClassName(): ClassName = representativeUpperBou
   .classDescriptor()
   .asClassName()
 
-fun PropertyReference.tangleParamNameOrNull(): String? {
+fun MemberPropertyReference.tangleParamNameOrNull(): String? {
```

Comment:
[`PropertyReference` was renamed `MemberPropertyReference` in Anvil 2.4.4](https://github.com/square/anvil/releases/tag/v2.4.4)